README.md

```diff
@@ -2,6 +2,7 @@
 The aim of this repository is to build a backend server for the Neuburger-Modell.
 ## Setup
 You need to install a mongodb on your machine. Follow https://docs.mongodb.com/getting-started/shell/installation/ and choose the guide for your OS. Before starting the neuburg-backend, make sure that the mongodb is running.
+For development: Before running the lint script or the flow type checker, execute the `flow-typed` script first.
```

Comment:
Why and what does it do? Some descrption would be nice :)

Reply:
It installs the flow type definitions for external libraries, so you can use e.g. classes from such libraries and annotate them with a correct flow type.

Reply:
kk, you can add it to the read me if you want. You can keep it very basic though.

Reply:
I added a small comment :)